packages/cli/test/utils/keymanagerTestRunners.ts

```diff
@@ -78,9 +79,10 @@ export function getKeymanagerTestRunner({args: {spawnCli}, afterEachCallbacks, d
 export async function expectKeys(keymanagerClient: Api, expectedPubkeys: string[], message: string): Promise<void> {
   const keys = await keymanagerClient.listKeys();
   ApiError.assert(keys);
-  expectDeepEquals(
-    keys.response.data,
-    expectedPubkeys.map((pubkey) => ({validatingPubkey: pubkey, derivationPath: "", readonly: false})),
-    message
-  );
+
+  // the order of keys isn't always deterministic so we can't use deep equal
```

Comment:
I would assume this change can be reverted since the keystore cache fix made the order of keys deterministic